.env.local

```diff
@@ -41,7 +41,7 @@ DB_ENTITIES=dist/src/modules/**/entities/**/*.entity{.ts,.js}
 DB_MIGRATIONS=dist/**/migrations/*{.ts,.js}
 DB_TYPE=postgres
 DB_SSL=true
-
+REDIS_PORT= 6379
```

Comment:
Why this? Your PR has no dependency on redis.

Reply:
@Cobidex, sorry about that sir. I forgot to remove that when I was trying to fix why the CI/CD fails. My new commit has that line removed, sir. 🙇